browse.html

```diff
@@ -36,72 +38,72 @@
         <h2>BROWSE ALL COCKTAILS</h2>
         <div class="cocktail-display-container">
             <div class="select-ingredients-container">
-            <div class="spirits">
-                <h4>SPIRITS</h4>
-                <div class="choose-spirits">
-                    <section>
-                        <input type="checkbox" id="vodka" name="VODKA">
-                        <label for="vodka">VODKA</label>
-                    </section>
-                    <section>
-                        <input type="checkbox" id="whiskey" name="WHISKEY">
-                        <label for="whiskey">WHISKEY</label>
-                    </section>
-                    <section>
-                        <input type="checkbox" id="gin" name="GIN">
-                        <label for="gin">GIN</label>
-                    </section>
-                    <section>
-                        <input type="checkbox" id="rum" name="RUM">
-                        <label for="rum">RUM</label>
-                    </section>
+                <div class="spirits">
+                    <h4>SPIRITS</h4>
+                    <div class="choose-spirits">
+                        <section>
+                            <input type="checkbox" id="vodka" name="VODKA">
+                            <label for="vodka">VODKA</label>
+                        </section>
+                        <section>
+                            <input type="checkbox" id="whiskey" name="WHISKEY">
+                            <label for="whiskey">WHISKEY</label>
+                        </section>
+                        <section>
+                            <input type="checkbox" id="gin" name="GIN">
+                            <label for="gin">GIN</label>
+                        </section>
+                        <section>
+                            <input type="checkbox" id="rum" name="RUM">
+                            <label for="rum">RUM</label>
+                        </section>
+                    </div>
```

Comment:
Notice in the changed code, the div closing immediately follows section closing tags...

Reply:
original </div> on line 58 / 61